index.html

```diff
@@ -22,7 +22,7 @@
 	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.css" />
 	<link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"
 		integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous" />
-	<link rel="stylesheet" href="styles/style.css" />
+	<link rel="stylesheet" href="styles/style.scss" />
```

Comment:
@cpickhardt15 you cannot include scss like that. This won't load css at all. Please add shake to regular css and commit. Thanks 🙏🏻